pkg/ccl/schemachangerccl/BUILD.bazel

```diff
@@ -7,15 +8,17 @@ go_test(
     srcs = [
         "main_test.go",
         "schemachanger_ccl_test.go",
+        ":test_gen_backup_non_ccl",  # keep
+        ":test_gen_ccl",  # keep
     ],
     data = glob(["testdata/**"]) + [
         "//pkg/sql/schemachanger:testdata",
     ],
     embed = [":schemachangerccl"],
     shard_count = 16,
+    tags = ["cpu:3"],
```

Comment:
just curious, what's this for? also, is the `size = "enormous"` still valid, now that the tests have been fragmented?

Reply:
This says that bazel should count this test as using 3 cores. If you have 16 cores, it means only run 5 of these in parallel. With these multi-node clusters, I find this is important. Otherwise the load average goes crazy and we don't see much benefit. In general, we need to be careful to set these in more places. Not having set these is part of the reason the build in regular CI felt like it stress. 

The test is still slow because the `Backup` test remains slow. I think leaving this as `size = "enormous"` is maybe too much, but I'm going to leave it for now. It only affects the timeout.